proparse/src/main/java/org/prorefactor/treeparser01/TP01Support.java

```diff
@@ -516,17 +528,17 @@ public void widattr(AST widAST, AST idAST, ContextQualifier cq) throws TreeParse
           result.variable.noteReference(cq);
         }
       }
-    } else if (idAST.getType() == NodeTypes.Field_ref) {
+    } else if (idAST.getType() == ProParserTokenTypes.Field_ref) {
       // Reference to a static field
       JPNode idNode = (JPNode) idAST;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unnecessary cast to "JPNode". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1905)